core/domain/email_manager.py

```diff
@@ -299,11 +367,39 @@ def send_post_signup_email(user_id):
 
 
 def require_valid_intent(intent):
+    """Checks if the given intent is valid, raises an exception if it is not.
+
+    Args:
+        intent: str. The intent string (cause/purpose) of the email.
+
+    Returns:
+        None
+
+    Raises:
+        Exception: The given intent did not match an entry in
+                    feconf.VALID_MODERATOR_ACTIONS
+    """
+
     if intent not in feconf.VALID_MODERATOR_ACTIONS:
         raise Exception('Unrecognized email intent: %s' % intent)
 
 
 def _get_email_config(intent):
+    """Return the default body for the email type
+        matching the given moderator action intent.
+
+    Args:
+        intent: str. The intent string (cause/purpose) of the email.
+
+    Returns:
+        str: The default body for the email type
```

Comment:
Done

For this, the reason I had used : instead of . in the Returns block is that PyCharm (IntelliJ) is then able to understand the return type and include it in the function information. The inferred type with "str." is "Any", but with "str:" it is "str".

It makes sense though that this doesn't match with the project's formatting conventions, I've changed the : to . here and in the other returns blocks.